core/server/common/src/test/java/alluxio/master/journal/JournalUtilsTest.java

```diff
@@ -75,6 +85,92 @@ public void checkpointAndRestoreComponents() throws Exception {
     components.forEach(c -> assertEquals(1, c.getNumEntriesProcessed()));
   }
 
+  @Test
+  public void noEntryTest() throws IOException, InterruptedException {
+    testEntries(0);
+  }
+
+  @Test
+  public void oneEntryTest() throws IOException, InterruptedException {
```

Comment:
do you think it makes sense to test taking multiple checkpoints in one test? This verifies taking one checkpoint doesn't break the system so it cannot take the next.

Reply:
`EmbeddedJournalIntegrationTestFaultTolerance#snapshotTransferLoad` tests to ensure that multiple snapshot are sent in succession.